src/azure-cli/azure/cli/command_modules/resource/_formatters.py

```diff
@@ -315,7 +315,7 @@ def _format_property_modify(builder, before, after, children, indent_level):
         # Space before =>
         if _is_non_empty_object(before):
             builder.append_line()
-            _format_indent(indent_level)
+            _format_indent(builder, indent_level)
```

Comment:
This is a good catch! Do you mind adding a test case to https://github.com/Azure/azure-cli/blob/9129fa866c86ef204ffa4bd5287692085b88ddbb/src/azure-cli/azure/cli/command_modules/resource/tests/latest/test_resource_formatters.py#L340?

Reply:
TBH I'm not that good in python to write such test. Would you mind to add it? I've added you as collaborator to my fork.

Reply:
Sounds good I'll push a commit.